tools/toc.sh

```diff
@@ -11,7 +11,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find * -name "*.md"); do
+for f in $(find . -path ./node_modules -prune -false -o -name "*.md"); do
```

Comment:
That expression seems interesting :smile:. It will also catch directories, so it'll probably break once we have a directory named `*.md`. It might be a good idea to add `-type f` after `-o`.

Also have you tested it on BSD `find` / macOS? I after checking the CI run and reading the macOS `find` manpage I _think_ it should work, but I don't have a machine to test on.